app/javascript/packs/homePage.jsx

```diff
@@ -73,7 +88,18 @@ let waitingForDataLoad = setTimeout(function dataLoadedCheck() {
   waitingForDataLoad = setTimeout(dataLoadedCheck, 40);
 }, 40);
 
-InstantClick.on('receive', (_url, body, title) => {
+InstantClick.on('change', () => {
+  const url = new URL(window.location);
```

Comment:
I originally had the feed rendering in the `InstantClick` on `receive` event, but this caused unintended behaviour. It did indeed work, but since the feed time frame prop of the `<Feed />` component changed, the current time frame tab would get updated with the time frame data of the time frame tab that was being hovered on.